backend/src/main/java/de/neuefische/henny/reisekasse/model/EventMember.java

```diff
@@ -10,8 +10,13 @@
 @AllArgsConstructor
 @Builder
 public class EventMember {
-
     private String username;
-    private double balance;
+    private String firstName;
+    private String lastName;
+    private int balance;
```

Comment:
you save the amount/balance on different places. You have to be carefully that the amounts are in sync. But it is for the mvp